nixos/modules/virtualisation/qemu-vm.nix

```
@@ -348,16 +352,41 @@ in
           '';
       };
 
+    virtualisation.shareExchangeDir =
+      mkOption {
+        default = true;
+        type = types.bool;
+        description =
+          ''
+            Share /tmp/shared and /tmp/xchg with the host.  These are used
+            for runInMachine, copy_from_{host,vm}, and coverage-data.
```

May want to use `<literal>` markup for easier reading, e.g.

```suggestion
            Share <literal>/tmp/shared</literal> and <literal>/tmp/xchg</literal> with the host.
            These are used for <literal>runInMachine</literal>, <literal>copy_from_{host,vm}</literal>,
            and <literal>coverage-data</literal>.
```

(Please check the manual builds after this, one can accidentlaly screw it up like I did [recently](https://github.com/NixOS/nixpkgs/pull/92240#issuecomment-653740926).)

---

@nh2: ... or go even further and use `<filename/>` and `<function/>`:

```suggestion
            Share <filename class="directory">/tmp/shared</filename> and
            <filename class="directory">/tmp/xchg</filename> with the host.
            These are used for <function>runInMachine</function>,
            <literal>copy_from_{host,vm}</literal>, and
            <literal>coverage-data</literal>.
```